.github/workflows/contracts.yml

```diff
@@ -211,7 +211,7 @@ jobs:
 
       - name: Install Solidity
         env:
-          SOLC_VERSION: 0.8.5 # according to solidity.version in hardhat.config.js
+          SOLC_VERSION: 0.8.9 # according to solidity.version in hardhat.config.ts
```

Comment:
`@threshold-network/solidity-contracts` uses `0.8.9` but we should be fine even with the most recent version, `0.8.17`. This is the version [used by `tbtc-v2`](https://github.com/keep-network/tbtc-v2/blob/f1d68c10a4b2276959e44d006b3ebd5c09da9876/solidity/contracts/bridge/Bridge.sol#L16).

Reply:
Worth noting, we should also bump pragma in `*.sol` files to make it work as expected.

Reply:
hm, the most recent version `0.8.17` didn't work out of the box even when I added multiple compilers in the hardhat.
```
Error HH606: The project cannot be compiled, see reasons below.
These files import other files that use a different and incompatible version of Solidity:
(here are a bunch of contracts listing different solidity versions 0.8.9 vs 0.8.17)
```

Reply:
I see. We would have to update contracts in https://github.com/threshold-network/solidity-contracts/tree/main/contracts/governance to use `pragma solidity ^0.8.9;`. Not something we have to worry about now.